common/web/gesture-recognizer/src/tools/build.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+#
+# Builds the include script for the current Keyman version.
+#
+
+# Exit on command failure and when using unset variables:
+set -eu
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../../../../../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+
+# This script runs from its own folder
+cd "$(dirname $THIS_SCRIPT)"
+
+################################ Main script ################################
+
+builder_describe "Testing-oriented tools for the Gesture Recognizer module of web-based Keyman OSKs" \
+  "clean" \
+  "build" \
+  ":fixture The HTML-element fixture and CSS fixture used for both user-testing and unit-testing" \
+  ":recorder The web page used for recording input sequences for use in unit-testing" \
+  ":test-module The TS library used to interface with the main gesture-recognizer module for tests"
```

Comment:
```suggestion
  ":fixture       The HTML-element fixture and CSS fixture used for both user-testing and unit-testing" \
  ":recorder      The web page used for recording input sequences for use in unit-testing" \
  ":test-module   The TS library used to interface with the main gesture-recognizer module for tests"
```

Just makes these easier to read (nit!)